packages/cli/src/commands/ecosystem.ts

```diff
@@ -18,50 +18,50 @@ ecosystemCommands
     const didKeyPairs = await getDIDKeyPairs({wallet});
     const didKeyPair = didKeyPairs[0];
 
-    await dockService.init({
+    await blockchainService.init({
       address: 'wss://knox-1.dock.io',
     });
 
-    await dockService.waitDockReady();
-    await dockService.ensureDockReady();
+    await blockchainService.waitDockReady();
```

Comment:
might be an Oleg question, but do we need a `waitCheqdReady`?

Reply:
yes, its missing here. I will rename this method to be generic, and inside it will wait for both cheqd and substrate to be ready